<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseWidgetClass</class>
 <widget class="QWidget" name="PauseWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PauseWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton_backMenu">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:transparent;
	background-image:url(./res/icon/home_white_64x.png);
	border:0px;
}

QPushButton:pressed
{
	background-color:transparent;
	background-image:url(./res/icon/home_black_64x.png);
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_restart">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:transparent;
	background-image:url(./res/icon/restart_white_64x.png);
	border:0px;
}

QPushButton:pressed
{
	background-color:transparent;
	background-image:url(./res/icon/restart_black_64x.png);
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_continue">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:transparent;
	background-image:url(./res/icon/continue_white_64x.png);
	border:0px;
}

QPushButton:pressed
{
	background-color:transparent;
	background-image:url(./res/icon/continue_black_64x.png);
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>675</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>pushButton_backMenu</zorder>
  <zorder>pushButton_restart</zorder>
  <zorder>pushButton_continue</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
